src/components/CalendarPopover/CalendarPopover.st.css

```diff
@@ -139,11 +133,6 @@
     left: 251px;
 }
 
-.title {
-    -st-mixin: TPAText(MainTextFont value(CurrentTitleFont),
-            MainTextColor value(CurrentTitleColor));
-}
-
```

Comment:
We'll need to leave this, so that when consumers pass a `string` it would be styled correctly.